packages/rfw/lib/flutter/core_widgets.dart

```diff
@@ -344,6 +344,15 @@ Map<String, LocalWidgetBuilder> get _coreWidgetsDefinitions => <String, LocalWid
     );
   },
 
+  'FractionallySizedBox': (BuildContext context, DataSource source) {
+    return FractionallySizedBox(
+      alignment: (ArgumentDecoders.alignment(source, ['alignment']) ?? Alignment.center).resolve(Directionality.of(context)),
```

Comment:
no need to resolve it, FractionallySizedBox accepts AlignmentGeometry

Reply:
thanks, updated in a following commit.